core/src/main/scala/com/nvidia/spark/rapids/tool/planparser/SQLPlanParser.scala

```diff
@@ -606,7 +608,8 @@ object SQLPlanParser extends Logging {
     } else if (buildSide == BuildSide.BuildRight) {
       joinType == JoinType.Inner || joinType == JoinType.Cross ||
         joinType == JoinType.LeftOuter || joinType == JoinType.LeftSemi ||
-        joinType == JoinType.LeftAnti || joinType == JoinType.FullOuter
+        joinType == JoinType.LeftAnti || joinType == JoinType.FullOuter ||
+        joinType == JoinType.ExistenceJoin
```

Comment:
Nit: we can define constant Sets of supported types instead of doing `match-case` and `if-else` blocks in different functions.
This can be part of the object `JoinType` and `BuildSide`.
 
```
supportedJoinType = Set(..,..,..)
supportedBuild: Map[string, Set] = {buildLeft => [..,..,..], buildRight => [..,.,,.,..]}
```

Reply:
Thanks @amahussein ! Updated the code. PTAL